tests/app/filter.test.ts

```diff
@@ -63,7 +63,21 @@ const selectIfSet = async (
   values?: string[]
 ): Promise<void> => {
   if (values !== undefined) {
-    await page.locator(selector).selectOption(values);
+    // Reset filters by unchecking all
+    const checkboxes = await page.$$(
+      `input[type="checkbox"][name="${selector}"]`
+    );
+    /* eslint-disable */
+    for (const checkbox of checkboxes) {
+      await checkbox.uncheck();
+    }
```

Comment:
```suggestion
   await Promises.all(checkboxes.map(box => box.uncheck())
```

Reply:
I tried this, too! But it will fail certain filter tests. Still looking into it.